### DISCUSSION TITLE
How do I update 1.4 to 1.7? (with cuda)

### DISCUSSION THREAD

**Discussion by slmslm330** [hacksider/Deep-Live-Cam#752] [General]
Hello

I tried to update to be able to have the additional camera options.

I managed to run with the new interface but it doesn't work during the preview and the following error appears

l erreur : onnxruntime.capi.onnxruntime_pybind11_state.InvalidProtobuf: [ONNXRuntimeError] : 7 : INVALID_PROTOBUF : Load model from C:\Users\me\Documents\Deep-Live-Cam\models\inswapper_128_fp16.onnx failed:Protobuf parsing failed.

Please help me

**Comment by hacksider** [comment]
Please use the template when reporting a bug/problem. Closing this, create a new one with the template that is given so we can easily help you.

**Comment by hacksider** [comment]
Anyway, based on the error, you just need to download inswapper_128_fp16.onnx manually

**Comment by slmslm330** [comment]
Okay, fine. So I'm going to look for this file which must be located on github I think. 
I hope I understood correctly
Then if I open a new post I have to go to the right section (if I understand correctly

Thank you

**Reply by KRSHH** [reply]
> Okay, fine. So I'm going to look for this file which must be located on github I think. I hope I understood correctly Then if I open a new post I have to go to the right section (if I understand correctly
> 
> Thank you

Direct links to the models are in the instructions in models folder.

**Reply by slmslm330** [reply]
Templates to create a discussion post or to solve my problem?

**Reply by KRSHH** [reply]
@slmslm330 You have to use the issue template in the issues section but instead, you have opened a discussion... 

There you go - https://github.com/hacksider/Deep-Live-Cam/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=

**Reply by slmslm330** [reply]
Okay, thank you.

I'll do it next time.

Thanks for the file, it works in v1. 7 Now
